src/hex_pb_names.erl

```diff
@@ -198,20 +199,24 @@ decode_msg(Bin, MsgName, Opts) when is_binary(Bin) ->
 -ifdef('OTP_RELEASE').
 decode_msg_1_catch(Bin, MsgName, TrUserData) ->
     try decode_msg_2_doit(MsgName, Bin, TrUserData)
-    catch Class:Reason:StackTrace -> error({gpb_error,{decoding_failure, {Bin, MsgName, {Class, Reason, StackTrace}}}})
+    catch
+         ?WITH_STACKTRACE(Class,Reason,StackTrace)
```

Comment:
I prefer the version with `->` in the `.erl` file, though it is a bit more verbose (but it doesn't feel so "strange").

e.g.

```
%hrl
-define(WITH_STACKTRACE(Class, Reason, _), Class:Reason).
-define(GET_STACKTRACE(_), erlang:get_stacktrace()).
```

then

```
%erl
catch
    ?CLASS_REASON_STACKTRACE(Class, Reason, StackTrace) ->
      error({gpb_error,{decoding_failure, {Bin, MsgName, {Class, Reason, ?GET_STACKTRACE(StackTrace)}}}})
```

(you can guess how it'd go for OTP 21+)